.github/workflows/rollback.yml

```diff
@@ -94,61 +110,63 @@ jobs:
             regions: "cn-north-1 cn-northwest-1"
     steps:
       - name: Configure AWS Credentials
-        if: matrix.type == 'global'
+        if: ${{ matrix.type == 'global' && github.event.inputs.skip-ssm-rollback == 'false' }}
```

Comment:
Is it possible to add this as a precondition to the job rather than have it in each step?

Reply:
Unfortunately not. If I have it at the job level, the job itself will count as cancelled and the dependent jobs like `all-checked` will treat that as a failure.